InternxtDesktop/AppDelegate.swift

```diff
@@ -453,5 +446,21 @@ class AppDelegate: NSObject, NSApplicationDelegate {
             display()
         }
         self.scheduledManager.resumeBackupScheduler()
+        self.checkRefreshToken()
+    }
+    
+    private func checkRefreshToken(){
```

Comment:
We already have 2 systems that performs tokens refresh, we need to add this logic to those systems, they are:

https://github.com/internxt/drive-desktop-macos/blob/main/SyncExtension/FileProviderExtension.swift#L69
https://github.com/internxt/drive-desktop-macos/blob/main/InternxtDesktop/AppDelegate.swift#L240